templates/paperless-ng.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>paperless-ng</Name>
+  <Repository>jonaswinkler/paperless-ng</Repository>
+  <Registry>https://hub.docker.com/r/jonaswinkler/paperless-ng/</Registry>
+  <Network>bridge</Network>
+  <Shell>bash</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/jonaswinkler/paperless-ng</Project>
+  <Overview>Index and archive all of your scanned paper documents.&#xD;
+[br][br]&#xD;
+Create a user account after container is created i.e. from the Unraid Docker UI, click the paperless-ng icon and choose console. Enter "python3.7 manage.py createsuperuser" in the prompt then and follow instructions.  [br][br]&#xD;
+Paperless-ng Documentation: https://paperless-ng.readthedocs.io/en/latest/[br]&#xD;
+Additional Template Variables: https://paperless-ng.readthedocs.io/en/latest/configuration.html</Overview>
```

Comment:
Might be worth converting this to just use markdown syntax, as that is the way forward.

Reply:
Sounds good. Is there any reference how this markdown syntax in the templates look like? Couldn't find examples in the repository ether.

Reply:
Also, what is the difference between the `Overview` and `Description` tag?

Reply:
Not really, it is a thing with the 6.9. I will look into updating the others later. 
```md
# This is a test

| markdown | table |
| --- | --- |
| yes | no
```
gives 
![image](https://user-images.githubusercontent.com/7110194/101981067-7f9b0d80-3c6a-11eb-8f67-cf851b53b9c0.png)
So its not fully implemented.

Reply:
> Also, what is the difference between the `Overview` and `Description` tag?

Overview gets used, unlike description

Reply:
I played around a little bit with the markdown syntax in the template and I'm not happy with it. I suggest to postpone this change until 6.9 is out. I'm willing to change this template myself then. For now, let's stick to the current state.